content-webapp/src/main/webapp/news-details/components/ExoNewsEditPublishingDrawer.vue

```diff
@@ -99,6 +99,8 @@
 </template>
 
 <script>
+import {NewsObjectType, NewsUpdateType} from '../../js/newsUtils';
```

Comment:
same for previous feedback of [ExoNewsDetails.vue](https://github.com/Meeds-io/content/pull/24/files#diff-cfe3af3815f6bc5b4301f41f556a356cb8f1611c90c27be4037f05a30d5744da)